tests/integration/test_patches.py

```diff
@@ -17,18 +17,23 @@ def git_repo(tmpdir):
     source-git repo.
     """
     repo = git.Repo.init(tmpdir)
-    shutil.copytree(
-        src="tests/data/patches/previous/",
-        dst=repo.working_tree_dir,
-        dirs_exist_ok=True,
-    )
+    try:
+        # dirs_exist_ok parameter doesn't exist in python < 3.8
+        shutil.copytree(
+            src="tests/data/patches/previous/",
+            dst=repo.working_tree_dir,
+        )
+    except FileExistsError:
+        pass
```

Comment:
This won't copy anything, if any of the dirs in the destination path exists, which is not what we want.

Reply:
fair enough